tools/opcode_metadata/cpython.py

```
@@ -0,0 +1,27 @@
+    CPYTHON_ROOT = pathlib.Path(os.environ["CPYTHON_ROOT"]).expanduser().resolve()
+except KeyError:
+    raise ValueError("Missing environment variable 'CPYTHON_ROOT'")
+
+CPYTHON_TOOLS_LIB = CPYTHON_ROOT / "Tools" / "cases_generator"
+
+
+if (path := CPYTHON_TOOLS_LIB.as_posix()) not in sys.path:
+    sys.path.append(path)
+
+
+from analyzer import SKIP_PROPERTIES, Analysis, Family, Properties, analyze_files
```

Flake8 (7.3.0)

[error] 'analyzer.SKIP_PROPERTIES' imported but unused

(F401)